CHANGES.rst

```diff
@@ -1,6 +1,14 @@
 CHANGELOG
 =========
 
+2.2.1
```

Comment:
change to unreleased

bumpversion takes care of stamping new version. It'll replace 2.2.0 with 2.2.1 if it won't find the unreleased word (see .bumpversion.cfg)

Reply:
I fixed that, and also rebased onto current `main`.